control-plane/tests/test_service_persistence_ingest_case_lifecycle.py

```
@@ -771,6 +771,58 @@ def test_service_extends_promoted_wazuh_alert_with_existing_case_linkage(
+            RuntimeConfig(postgres_dsn="postgresql://control-plane.local/aegisops"),
+            store=store,
+        )
+        adapter = WazuhAlertAdapter()
+        payload = _load_wazuh_fixture("agent-origin-alert.json")
+        created = service.ingest_native_detection_record(
+            adapter,
+            adapter.build_native_detection_record(payload),
+        )
+        native_evidence = next(
+            evidence
+            for evidence in store.list(EvidenceRecord)
```

Ruff (0.15.12)

[error] Undefined name `EvidenceRecord`

(F821)